### PR TITLE
Lights

### PR DESCRIPTION
**Pull request by nem0**

### CODE REVIEW

File: src/graphics/render_scene.cpp [hunk 1]
```diff
@@ -250,23 +251,27 @@ namespace Lumix
...
-
-	struct Light
+	
+	struct PointLight
 	{
-		enum class Type : int32_t
-		{
-			DIRECTIONAL
-		};
+		bool m_is_empty;
```
Comment (tluqo) [inline src/graphics/render_scene.cpp]: I would shift bool variables towards the end of the struct

File: src/graphics/render_scene.cpp [hunk 2]
```diff
@@ -250,23 +251,27 @@ namespace Lumix
...
+		Vec4 m_color;
+		float m_intensity;
+		float m_range;
+		Entity m_entity;
+	};
+
 
+	struct GlobalLight
+	{
+		bool m_is_empty;
```
Comment (tluqo) [inline src/graphics/render_scene.cpp]: I would shift bool variables towards the end of the struct

**Comment by tluqo** [comment]
Ship it!

File: src/graphics/render_scene.cpp (hunk 1 above)
Comment (nem0) [inline src/graphics/render_scene.cpp]: bool variables are no more - fixed

File: src/graphics/render_scene.cpp (hunk 2 above)
Comment (nem0) [inline src/graphics/render_scene.cpp]: bool variables are no more - fixed

**Pull request merged**